### ISSUE TITLE
Extended scalars doesn'nt work as described

### ISSUE THREAD

**Issue by Thinkenterprise** [graphql-java-kickstart/graphql-spring-boot#629]
I define a scalar type `scalar Date` in my schema. Then I configured extended scalars over the configuration 

```
graphql:
  extended-scalars: Date 
```

and expect that the graphql spring boot starter autoconfiguration register the new type automatically. 

But I got the following Exception 

```
Caused by: graphql.kickstart.tools.SchemaClassScannerError: Expected a user-defined GraphQL scalar type with name 'Date' but found none!
```

I added the library

```
   <dependency>
	<groupId>com.graphql-java</groupId>
	<artifactId>graphql-java-extended-scalars</artifactId>
	<version>15.0.0</version>
   </dependency>
```

But I have still the same problem because I still have no auto-configuration.
I miss an auto-configuration like with this library.  

```
<dependency>
	<groupId>com.zhokhov.graphql</groupId>
	<artifactId>graphql-datetime-spring-boot-starter</artifactId>
	<version>4.0.0</version>
</dependency>
```

Or am I doing something wrong?

**Comment by oliemansm** [comment]
You shouldn't have to define that extended-scalars dependency. The one you're listing here is actually older than the one being pulled in by this library. 

The GraphQLExtendedScalarsInitializer should be auto-configuring them.

**Comment by Thinkenterprise** [comment]
I used version **11.0.0**. The feature is available with **11.1.0**. That was my fault!!
